src/main/java/net/sf/jabref/logic/util/io/SimpleFileListEntry.java

```diff
@@ -0,0 +1,94 @@
+/*  Copyright (C) 2015 JabRef contributors.
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+package net.sf.jabref.logic.util.io;
+
+import java.util.List;
+
+/**
+ * This class represents a simplified version of the file link for a Bibtex entry.
+ */
+public class SimpleFileListEntry {
```

Comment:
The `SimpleFileList` class right now has two purposes. For one, it contains the list of linked files, and secondly, it has the `setContent` method which parses a string to proper list of `SimpleFileListEntries`. So the idea was to move the latter functionality to a separate class. In detail:
1. Move `setContent` to a new class (maybe called `FileListBuilder` or `FileListFactory`) or move it to `FileUtil` as you want. In order to make the method work in its new home, you need to return the generated list of file entries (instead of writing it to a private field).
2. Now nothing is left in `SimpleFileList`. So it can be replaced by `List<SimpleFileListEntry>`.